back/spec/lib/carrierwave_temp_remote_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CarrierwaveTempRemote do
+  subject { described_class }
+  let(:model) { create(:project) }
+  let(:remote_url) { 'https://cl2-seed-and-template-assets.s3.eu-central-1.amazonaws.com/images/brainstorming_graphic.png' }
+
+  describe '.save_urls' do
+    it 'saves remote url to model' do
+      image_assignments = { remote_header_bg_url: remote_url }
+      subject.save_urls(model, image_assignments)
+      expect(model['header_bg']).to eq(remote_url)
+    end
+
+    it 'downloads and sets image from stored temp remote url when it finishes background job', active_job_inline_adapter: true do
```

Comment:
Didn't know about `active_job_inline_adapter`, TIL, thanks!

Reply:
JFTR it's a custom thing in our project https://github.com/CitizenLabDotCo/citizenlab/blob/893855776552bfa80c1ac16584e796a32587f8dc/back/spec/spec_helper.rb#L193